src/js/config/Config.js

```diff
@@ -11,6 +11,7 @@ var Config = {
   secretsAPIPrefix: '/secrets/v1',
   delayAfterErrorCount: 5,
   documentationURI: 'https://dcos.io/docs',
+  marathonJSONDocsURI: 'https://mesosphere.github.io/marathon/docs/generated/api.html#v2_apps_post',
```

Comment:
The URI isn't actually to the home of the docs, but to a modal in the docs. Maybe we should have a "generic" URI here. Also, maybe we shouldn't put it in the config because it's completely related to the services UI.